pkg/topology/topology.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package topology
+
+import (
+	"errors"
+	"math/rand"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/p2p"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var ErrNotFound = errors.New("no peer found")
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
It would be more lear why the seed is called if it would be in `pkg/topology/full/full.go`, as there is rand.Intn.

Reply:
copy paste leftovers :D